module2/lessons/intro_to_building_a_rails_api.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Intro to Building an API in Rails
+length: 60
+layout: page
+---
+
+## Learning Goals
+
+- Understand how an API works at a conceptual level
+- Understand REST and how it influences API development
+- Understand what makes a valid JSON data structure
+
+
+## Reviewing REST
```

Comment:
We have a lesson on REST in the following week so I'd opt to not focus too much on what it is so that we can introduce it more fully later. A bit of exposure makes sense, but at this point they've had no intro so it wouldn't be "Reviewing"

Reply:
Noted. Thanks!